Profiles/ext-RelationHypersensitivityIntolerance.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="ext-RelationHypersensitivity" />
-  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationHypersensitivity" />
+  <id value="ext-RelationHypersensitivityIntolerance" />
```

Comment:
This extension should be renamed to ext-Reaction.RelationHypersensitivityIntolerance, since the RelationHypersensitivityIntolerance concept is only part of the Reaction building block.